readme.md

```diff
@@ -120,7 +120,7 @@ $ ava --help
   test.js test-*.js test/*.js
 ```
 
-Files in directories named `fixtures` and `helpers` are ignored, as well as files starting with `_`. This can be useful for having helpers in the same directory as your test files.
+Directories are recursive by default. Files in directories named `fixtures` and `helpers` are ignored, as well as files starting with `_`. This can be useful for having helpers in the same directory as your test files.
```

Comment:
Not totally happy with this, but something like:

> By default, AVA run tests named `test.js` or starting with `test-` in root and any tests recursively in the `test` directory.

You also need to update the globbing pattern in the CLI help output (and the identical output in the readme).